Jenkinsfile

```diff
@@ -83,6 +83,7 @@ pipeline {
 
                     finally{
                         sh "build/run go get -u -f  github.com/jstemmer/go-junit-report"
+                        sh "journalctl -u kubelet > _output/tests/kubelet.log"
```

Comment:
wouldn't we want this per k8s version? should this be done in the `RunIntegrationTest` function instead?

Reply:
i knew i was missing something...